src/utils/viewer/viewer.cpp

```diff
@@ -687,7 +687,7 @@ void addSensors(esp::agent::AgentConfiguration& agentConfig, bool isOrtho) {
     }
     spec->position = {0.0f, rgbSensorHeight, 0.0f};
     spec->orientation = {0, 0, 0};
-    spec->resolution = esp::vec2i(viewportSize[1], viewportSize[0]);
+    spec->resolution = Mn::Vector2i(viewportSize[1], viewportSize[0]);
```

Comment:
```suggestion
    spec->resolution = viewportSize.flipped();
```

Same in the cases below.